### PR TITLE
Create getter and setter for  MarketplaceService.Stubbed

### PR DESCRIPTION
**Pull request by migueleliasweb**
- Regenerated interfaces
- New `MarketplaceService.GetStubbed` and `MarketplaceService.SetStubbed`

### CODE REVIEW

File: github/apps_marketplace.go [hunk 1]
```diff
@@ -75,6 +75,14 @@ type MarketplacePlanAccount struct {
 	MarketplacePendingChange *MarketplacePendingChange `json:"marketplace_pending_change,omitempty"`
 }
 
+func (s *MarketplaceService) SetStubbed(stubbed bool) {
```
Comment (gmlewis) [inline github/apps_marketplace.go:78]: These two new methods need standard godoc-style comment strings.

Comment (migueleliasweb) [reply github/apps_marketplace.go:78]: I don't know how I managed to miss this 😔

Comment (migueleliasweb) [reply github/apps_marketplace.go:81]: Done, added the doc strings

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @migueleliasweb .
Sorry to be a pain, but please make the comments complete sentences per idiomatic Go style for the auto-generated godocs.

File: github/apps_marketplace.go [hunk 2]
```diff
@@ -75,6 +75,20 @@ type MarketplacePlanAccount struct {
...
 }
 
+// SetStubbed configures MarketplaceService Stubbed property
+//
+// GitHub API dos: https://docs.github.com/en/developers/github-marketplace/testing-your-app#testing-apis
+func (s *MarketplaceService) SetStubbed(stubbed bool) {
+	s.Stubbed = stubbed
+}
+
+// GetStubbed returns MarketplaceService Stubbed property
```
[the remark this answers is not in the document]
Comment (migueleliasweb) [reply github/apps_marketplace.go]: Added :)

File: github/apps_marketplace.go [hunk 3]
```diff
@@ -75,6 +75,20 @@ type MarketplacePlanAccount struct {
 	MarketplacePendingChange *MarketplacePendingChange `json:"marketplace_pending_change,omitempty"`
 }
 
+// SetStubbed configures MarketplaceService Stubbed property
```
[the remark this answers is not in the document]
Comment (migueleliasweb) [reply github/apps_marketplace.go]: Added :)

[the remark this answers is not in the document]
Comment (gmlewis) [reply github/apps_marketplace.go]: Sorry again to be a pain, but can you please add the ending periods in both cases as shown in the suggested changes?

[the remark this answers is not in the document]
Comment (migueleliasweb) [reply github/apps_marketplace.go]: Sure, no problem!

[the remark this answers is not in the document]
Comment (migueleliasweb) [reply github/apps_marketplace.go]: Done!

**Review by gmlewis — APPROVED** [review]
LGTM, thank you, @migueleliasweb .
Merging.

**Pull request merged**